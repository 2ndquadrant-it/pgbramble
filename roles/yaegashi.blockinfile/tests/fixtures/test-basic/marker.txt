<h1>111 222 333</h1>
