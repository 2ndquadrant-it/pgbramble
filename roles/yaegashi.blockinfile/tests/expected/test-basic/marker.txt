<h1>111 222 333</h1>
<!-- BEGIN ANSIBLE MANAGED BLOCK -->
<h2>aaa bbb ccc</h2>
<h2>AAA BBB CCC</h2>
<!-- END ANSIBLE MANAGED BLOCK -->
